parlai/crowdsourcing/tasks/turn_annotations_static/hydra_configs/conf/example.yaml

```diff
@@ -11,20 +11,14 @@ mephisto:
     extra_source_dir: ${task_dir}/webapp/src/static
     units_per_assignment: 5
     onboarding_qualification: turn_annotations_static
-    annotation_buckets: ${task_dir}/task_config/annotation_buckets.json
+    annotations_config_path: ${task_dir}/task_config/annotations_config.json
```

Comment:
Hmm, here `annotations_config.json` has an `s` at the end of `annotations`, but in `parlai/crowdsourcing/tasks/turn_annotations_static/analysis/compile_results.py` it doesn't